srcpkgs/armadillo/template

```diff
@@ -10,7 +10,7 @@ maintainer="Julien Dehos <dehos@lisic.univ-littoral.fr>"
 license="Apache-2.0"
 homepage="http://arma.sourceforge.net/"
 distfiles="${SOURCEFORGE_SITE}/arma/${pkgname}-${version}.tar.xz"
-checksum=5f00c735e97fc23c910a225d1b6db9541af76022cc74f9b8b327dfa347926b40
+checksum=b942d10bbd0b6a30f5bbe63cb7e8820f64227adcf3a2ba4bb1bc785526bb957b
 
 # automatically link to openblas (so you just have to compile with
 # "-larmadillo" instead of "-larmadillo -lopenblas")
```

Comment:
move configure_args to directly below build_style

also shouldn't it be ${XBPS_CROSS_BASE}/usr/lib/libopenblas.so ? this current configuration makes cross link against the host machine not the cross machine from what i can tell

Reply:
Thanks. Actually, there is no need specify this at all. CMake picks it up by itself.